src/models/users/useUserMetadataSchemas.js

```diff
@@ -31,6 +34,28 @@ export default function useUserMetadataSchemas(displayedUserId) {
             }),
           ]
         : [];
+      const intelligentAgentFields = map(
```

Comment:
is it just me or was this exact code in the last PR too?

Reply:
Very similar, but not exact.
![Screen Shot 2022-05-12 at 10 22 20 AM](https://user-images.githubusercontent.com/2775448/168132940-beac24ad-b26e-4df8-8e6f-a6df98751413.png)